cmake/OssiaDeps.cmake

```diff
@@ -166,3 +167,10 @@ if(MSVC)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS)
   add_definitions(-D_SCL_SECURE_NO_WARNINGS)
 endif()
+
+include(FetchContent)
+FetchContent_Declare(
+    rapidfuzz
+    SOURCE_DIR ${CMAKE_SOURCE_DIR}/3rdparty/rapidfuzz-cpp
```

Comment:
should put it in quotes in case of people building in paths with spaecs

Reply:
fixed with 4eed7129